CDCgov/prime-reportstream
# 10839: elims parse as 27

This PR completes the required ELIMS updates getting the messages to match exactly

Test Steps:
1. Update the DEV_ELIMS receiver to be active
2. Send any of the elims_* messages from the integration tests from the ignore.ignore-elr-elims sender
3. Verify that the messages match exactly (barring the changes the ELIMS team will make)

## Changes
- Changes the HL7 reader to first try parsing an ORU message as v27
- Updates the base ORU FHIR->HL7 schema to encode fields as CWE where they are CWE in the the NIST 2.5.1 spec

## Checklist

### Testing

### Process

## Linked Issues
- #10839

## To Be Done


## Specific Security-related subjects a reviewer should pay specific attention to
